pkg/webui/account/containers/sessions-table/index.js

```diff
@@ -54,7 +54,10 @@ const UserSessionsTable = props => {
   const { pageSize, user, handleDeleteSession, sessionId } = props
   const dispatch = useDispatch()
 
-  const getSessions = React.useCallback(filters => getUserSessionsList(filters, user), [user])
+  const getSessions = React.useCallback(
+    filters => getUserSessionsList(user, { ...filters }),
```

Comment:
```suggestion
    filters => getUserSessionsList(user, filters),
```
Why do you clone the object again via spread?